lib/templates/auth.store.js

```diff
@@ -146,7 +146,9 @@ export default {
       <% } %>
 
       // Fetch authenticated user
+      <% if (options.user.fetchUser) { %>
```

Comment:
Thanks for this PR. What's your idea if we conditionally remove fetchUser action from store too when this option is `false`?

Reply:
In my project I don't keep a state of user, token is enough therefore I wanted to make this conditional